docs/proposal/bypass_zh.md

```diff
@@ -0,0 +1,76 @@
+## 背景
+
+目前，如果服务之间在开启服务网格的情况下通信失败，很难判断问题是因为服务网格接管流量造成的，还是服务本身存在错误。 因此，我们提供了将网格服务从通信流量路径中删除的功能，以查明问题是由服务网格还是应用程序本身的错误引起的。
+
+## 功能设计
+
+bypass是一种通用网格功能，用于从流量路径中删除 sidecar/kmesh 网格服务。 下面两种场景介绍了启用bypass功能后的流量路径。
+
+### 场景一：sidecar
+
+下面两张图中，在集群中启用了sidecar来接管流量。 图一为启用bypass功能前的流量路径。 从中我们可以看到，流量路径是服务A->sidecar->sidecar->服务B，当服务A和服务B通信失败时，从黑盒层面，用户无法判断是否是 sidecar 或服务本身存在问题。 图二中，开启了pod1的bypass功能，从而pod1中的sidecar会被从流量路径中移除。 如果此时服务A和服务B能够正常通信，则证明sidecar存在问题。
+
+![alt text](pics/sidecar_pre_bypass.svg)
+
+![alt text](pics/sidecar_bypass.svg)
+
+### 场景二：kmesh
+
+在图一中，流量路径是服务A->kmesh->服务B。在图二中，开启了bypass功能后，会在kmesh的ebpf程序中判断bypass功能已经开启，从而kmesh不会接管服务A发出的数据，流量路径是服务A->服务B。如果此时服务A和服务B能够正常通信，则证明kmesh存在问题。
+
+![alt text](pics/kmesh_pre_bypass.svg)
+
+![alt text](pics/kmesh_bypass.svg)
+
+
+## Bypass的数据结构
+
+```c
+struct bpf_map_def SEC("maps") bypass_fliter_map = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(__u32),
+    .value_size = sizeof(__u8),
+    .max_entries = MAX_ENTRIES,
```

Comment:
The pod number in a node is limited, regularly it is 110. So we can set it like 256 or 512.

Reply:
> The pod number in a node is limited, regularly it is 110. So we can set it like 256 or 512.

Yes，I have set it to 256